json-format.md

```diff
@@ -92,9 +92,12 @@ respective CloudEvents type when the mapping rules are fulfilled.
 ### 2.3. Mapping Data
```

Comment:
Terrible nit, but I think this should be in section 3 now, because we've removed `data` from the attributes, and section 2 is "Attributes".

Reply:
That would also put this text closer to the text in section 3.1 'Special Handling of "data"'.